src/main/java/org/sagebionetworks/web/client/widget/table/v2/TotalVisibleResultsWidget.java

```diff
@@ -48,7 +51,8 @@ public void configure(View viewEntity) {
 		// Currently, the only case where we can reliably get this info is for the current version of a Dataset.
 		// Other cases will need a new service, tracked by PLFM-7046
 		if (viewEntity instanceof Dataset && viewEntity.getIsLatestVersion()) {
-			this.totalNumberOfResults = ((Dataset) viewEntity).getItems().size();
+			List<DatasetItem> datasetItems = ((Dataset) viewEntity).getItems();
+			this.totalNumberOfResults = datasetItems != null ? datasetItems.size() : 0;
```

Comment:
Unrelated fix, just need a null check here (null value is possible but unlikely).